tools/generate-changelog.py

```diff
@@ -37,7 +39,8 @@
 # keep the list short to not increase the time it takes
 # to generate changelogs
 pkg_allowed_list = [
-    'distro-info-data', # is removed during hook
```

Comment:
i belive this was broken before on the CI, but for some reason wasn't hit

Reply:
ok, it is a bit strange though that the list here is quite different to the one for core24 (we have there dpkg, openssl, and base-files).

Reply:
I already added to my TODO to look more into this - I have a sneaking suspecion we might be not removing the packages we expect since this happens